src/plugins/index_pattern_management/public/plugin.ts

```diff
@@ -167,13 +167,17 @@ export class IndexPatternManagementPlugin
       },
     });
 
-    core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.settingsAndSetup, [
-      {
-        id: IPM_APP_ID,
-        title: sectionsHeader,
-        order: 400,
-      },
-    ]);
+    core.getStartServices().then(([coreStart]) => {
+      if (!coreStart.application.capabilities.workspaces.enabled) {
```

Comment:
Would be nice to leave a comment to elaborate this if condition

Reply:
Sure, added.